bulbs/content/tasks.py

```diff
@@ -34,3 +33,21 @@ def index_feature_type_content(featuretype_pk):
     featuretype = FeatureType.objects.get(pk=featuretype_pk)
     for content in featuretype.content_set.all():
         content.index()
+
+
+@shared_task(default_retry_delay=5)
+def update_feature_type_rates(featuretype_pk):
+    from bulbs.contributions.models import ContributorRole, FeatureTypeRate, FEATURETYPE
```

Comment:
it is. It's somewhat necessary for celery tasks however due to circular imports. Since tasks are commonly called by objects they intend to reference in separate threads. Low memory cost for that same reason however.